### DISCUSSION TITLE
在启动器错误窗口中提示清理.NET程序集缓存

### DISCUSSION THREAD

**Discussion by fireboy637** [Meloong-Git/PCL#5107] [讨论]
在我的使用中发现，大部分时候PCL2在同一个地方反复出现相同的错误时，使用工具（我使用DISM++）清除.NET程序集缓存就可以消除这些错误，因此建议在启动器错误提示中添加类似字段

**Comment by wuliaodexiaoluo** [comment]
>提示清理.NET程序集缓存 

清理缓存并不一定一直会带来正面反馈，不建议加这个
并且并不是所有人都有能力使用工具，加这个并不一定会解决问题，反而有可能带来各种无效反馈（（（

**Reply by fireboy637** [reply]
其实大部分能清理系统垃圾的软件/功能（例如火绒等）都带有.NET清理项
这个提示作为一个处理方案，目的是减少或防止这类可解决问题被反馈为BUG，不过即使不手动清理，缓存也会被自动清理

**Reply by wuliaodexiaoluo** [reply]
>相同的错误

我想知道是什么错误，这有助于我们评估到底有没有必要加

**Comment by MoYuan-CN** [comment]
个人认为不应当在启动器崩溃提示中加入技术性提示内容

**Comment by LTCatt** [comment]
> 大部分时候PCL2在同一个地方反复出现相同的错误时

我觉得这个你可以直接交个 bug 反馈…………
